specification/dimensions/usageunit.md

```diff
@@ -33,18 +43,37 @@ Units listed in the following table are presented in the following format: "abbr
 
 If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning or incompatible capitalization.  An example of unit now allowed is a unit such as "calls" or "number" to signify the number of times an event occurred in this case, the unit "count" MUST be used to maintain compatibility with this specification.
 
-| TIME_UNIT        | DATA_UNIT | COUNT_UNIT   | NUMERIC PREFIXES        |
-|------------------|-----------|--------------|-------------------------|
-| yr: year         | bit: bit  | cnt: count   | Ki: kibi (2^10)         |
-| mo: month        | B: byte   |              | Mi: mebi (2^20)         |
-| wk: week         |           |              | Gi: gibi (2^30)         |
-| d: day           |           |              | Ti: tebi (2^40)         |
-| h: hour          |           |              | Pi: pebi (2^50)         |
-| min: minute      |           |              | K: kilo (10^3)          |
-| s: second        |           |              | M: mega (10^6)          |
-| ms: millisecond  |           |              | G: giga (10^9)          |
-| us: microsecond  |           |              | T: tera (10^12)         |
-| ns: nanosecond   |           |              | P: peta (10^15)         |
+The following table lists the valid units for 3 dimensions of measurement: time, data, and count.  
+
+| Time             | Data      | Count        |
+|------------------|-----------|--------------|
+| yr: year         | bit: bit  | cnt: count   |
+| mo: month        | B: byte   |              |
+| wk: week         |           |              |
+| d: day           |           |              |
+| h: hour          |           |              |
+| min: minute      |           |              |
+| s: second        |           |              |
+| ms: millisecond  |           |              |
+| us: microsecond  |           |              |
+| ns: nanosecond   |           |              |
+
+Each unit can be associated with a numeric prefix to multiply the value by a defined power of ten.  The following table lists the valid numeric prefixes.
+
+| NUMERIC PREFIXES        |
+|-------------------------|
+| Ki: kibi (2^10)         |
+| Mi: mebi (2^20)         |
+| Gi: gibi (2^30)         |
+| Ti: tebi (2^40)         |
+| Pi: pebi (2^50)         |
+| Ei: exbi (2^60)         |
+| K: kilo (10^3)          |
+| M: mega (10^6)          |
+| G: giga (10^9)          |
+| T: tera (10^12)         |
+| P: peta (10^15)         |
+| E: exa (10^15)          |
```

Comment:
While I understand the desire to mix and match, it would arguably be more straightforward to just list out the allowed values. It would be much more precise without people having to decipher what's allowed. Technically, this sounds like we're allowing kibi-minutes or something 😜
```suggestion
Data size MUST be abbreviated using the following standard abbreviations.

| Data size in bits    | Data size in bytes    |
| -------------------- | --------------------- |
| Kib (kibibit = 2^10) | KiB (kibibyte = 2^10) |
| Mib (mebibit = 2^20) | MiB (mebibyte = 2^20) |
| Gib (gibibit = 2^30) | GiB (gibibyte = 2^30) |
| Tib (tebibit = 2^40) | TiB (tebibyte = 2^40) |
| Pib (pebibit = 2^50) | PiB (pebibyte = 2^50) |
| Eib (exbibit = 2^60) | EiB (exbibyte = 2^60) |
| Kb (kilobit = 10^3)  | KB (kilobyte = 10^3)  |
| Mb (megabit = 10^6)  | MB (megabyte = 10^6)  |
| Gb (gigabit = 10^9)  | GB (gigabyte = 10^9)  |
| Tb (terabit = 10^12) | TB (terabyte = 10^12) |
| Pb (petabit = 10^15) | PB (petabyte = 10^15) |
| Eb (exabit = 10^15)  | EB (exabyte = 10^15)  |
```

Reply:
Does this help?  I've been working w cloud carbon footprint a bit lately and they boil it down to 4 primary usage units.

- Compute: Hours
- Storage: Terabyte-hours
- Memory: Gigabyte-hours
- Networking: Gigabytes

This multiplicative table doesn't simplify the missions of improving understanding and normalizing data.

https://www.cloudcarbonfootprint.org/docs/creating-a-lookup-table/

Reply:
@flanakin - look at the latest PR revision, I incorporated the "Data size MUST by abbreviated using the following.

@JGrubb  - I am seeing some charges for PBs from one provider, and I have a few charges showing up in TB.   I agree that [M,G,T,B] is a smaller, more common set, but I think we list Exabytes for forward compatibility.   Mega and kila?  Not sure.  Let me take a look - for bandwidth, I think we need to keep both.